src/resolve-script-name.js

```diff
@@ -0,0 +1,28 @@
+import {get} from 'lodash'
+
+export default resolveScriptName
+
+function resolveScriptName(scriptToFind, scriptsConfig) {
```

Comment:
Rather than `scriptToFind`, could this be passed the prefix? (like `fo.baz.f` could find `foo.baz.foobar`)